### PR TITLE
Fixes highlighted by the "picking a CHERI allocator" paper.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2590,7 +2590,7 @@ @article{woodruff19chericoncentrate
       Roe, Michael and Neumann, Peter G. and Watson, Robert N. M. and
       Moore, Simon W.",
     title = "{CHERI} {Concentrate}: Practical Compressed Capabilities",
-    journal = {IEEE Transactions on Computers},
+    journal = {TCOM},
```
Comment (0152la) [inline softdev.bib:2593]: When I search "IEEE TCOM", I get suggested "Transactions on Communications". Can't find an appropriate acronym for "Transactions on Computers". If that's not an issue, we can go ahead with this acronym as is.

Comment (ltratt) [reply softdev.bib]: Oops! Re-expanded in 08ff05d.

**Comment by ltratt** [comment]
OK to squash?

**Comment by 0152la** [comment]
Whoops sorry; yes, please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**